ledger-service/http-json/src/main/scala/com/digitalasset/http/EndpointsCompanion.scala

```diff
@@ -317,7 +321,10 @@ object EndpointsCompanion {
       case ParticipantServerError(grpcStatus, d) =>
         grpcStatus.asAkkaHttpForJsonApi -> s"$grpcStatus${d.cata((": " + _), "")}"
       case ServerError(reason) =>
-        logger.error(s"Internal server error occured: $reason")
+        reason.fold(
+          logger.error(s"Internal server error occured", _),
+          msg => logger.error(s"Internal server error occured: $msg"),
+        )
```

Comment:
Here also the disjunction does not seem useful?

Reply:
I think I would prefer to get the stacktrace string from Throwable in apply of `InternalError`, so you don't have to think about it later on.